src/model/mongodb/mongodb-repository.ts

```diff
@@ -140,7 +140,8 @@ class MongoDBArticleRepository implements ArticleRepository {
     );
 
     if (!version) {
-      throw Error('Cannot find a matching article version');
+      logger.info(`Cannot find a matching article version (${identifier})`);
```

Comment:
potentially, now that not finding a version is expected, this should/could be logged at the controller level instead?